packages/code/src/main.rs

```diff
@@ -126,7 +226,7 @@ fn get_style(
     style
 }
 
-fn get_highlighted(code: &str, lang: &String, tm: &str) -> (String, Option<Color>) {
+fn get_highlighted_html(code: &str, lang: &String, tm: &str) -> (String, Option<Color>) {
     let ss = SyntaxSet::load_defaults_newlines();
     let ts = ThemeSet::load_defaults();
     let theme = match tm {
```

Comment:
We have two (to me) identical theme matches, and loading of `SyntaxSet` and `ThemeSet`, might be nicer to not do that in two places